src/style.css

```diff
@@ -1,3 +1,94 @@
+html {
+  font-size: 12px;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+
+  --link-color: #00acdc;
+}
+
+.header {
+  height: 60px;
+
+  /* width: 100%; */
+  display: flex;
+
+  /* align-items: center; */
+  justify-content: space-between;
+  padding: 0 50px;
+
+  /* box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05); */
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  display: block;
+  justify-content: space-between;
```

Comment:
redundant
```suggestion
  display: block;
```

Reply:
Do you need to remove these styles so that the work doesn't match the template from the figma ?